core/common-util/src/main/java/datawave/core/common/logging/ThreadConfigurableLogger.java

```diff
@@ -3,216 +3,132 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.apache.log4j.Appender;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
-import org.apache.log4j.Priority;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
-public class ThreadConfigurableLogger extends Logger {
+public class ThreadConfigurableLogger {
```

Comment:
Why did we stop extending a class here? Could we extend `org.apache.logging.log4j.Logger`?

Reply:
Logger is an interface in Log4j 2.x

Reply:
There's a `SimpleLogger` that might be analogous to the log4j 1 `Logger`. If that's not quite right, I'd keep looking around for some concrete class to extend that isn't as heavy as the log42 `Logger` interface. If we can't find something suitable, we could consider removing some usages of `ThreadConfigurableLogger` that aren't necessary.